pkg/api/sh.go

```diff
@@ -12,3 +22,33 @@ func ShellCommand(script string) (string, error) {
 
 	return string(out), nil
 }
+
+func Exec(command string) error {
+
+	if strings.TrimSpace(command) == "" {
+		return errors.New("command cannot be empty")
+	}
+
+	regularUser, err := userinfo.GetRegularUser()
+	if err != nil {
+		return err
+	}
+
+	uid, err := strconv.Atoi(regularUser.Uid)
+	if err != nil {
+		return err	
+	}
+
+	err = syscall.Setreuid(uid, uid)
+	if err != nil {
+		return err	
+	}
+
+	const executablesDirectory = "/usr/bin"
+	argv := strings.Split(command, " ")
+	argv[0] = filepath.Join(executablesDirectory, command)
+
+	err = syscall.Exec(argv[0], argv, os.Environ())
```

Comment:
Does it allow using pipes?

Reply:
No it doesn't. But should we support it in an exec call anyway? Exec is used for replacing the current process

Reply:
This function was meant to be used at the end of rules for executing the specified DE or Window Manager